src/chronicle_storage.erl

```diff
@@ -158,25 +170,26 @@ open(PublishSnapshotFun) ->
 
     publish(Storage5).
 
-open_logs(#storage{data_dir = DataDir} = Storage) ->
-    {Orphans, Sealed, Current} = find_logs(DataDir),
-
-    NewStorage0 = Storage#storage{meta = #{},
-                                  pending_meta = #{},
-                                  config = undefined,
-                                  committed_config = undefined,
-                                  pending_configs = [],
-                                  low_seqno = ?NO_SEQNO + 1,
-                                  high_seqno = ?NO_SEQNO,
-                                  pending_high_seqno = ?NO_SEQNO,
-                                  current_snapshot = no_snapshot,
-                                  extra_snapshots = [],
-                                  log_segments = []},
-
-    %% TODO: Be more robust when dealing with corrupt log files. As long as a
-    %% consistent state can be recovered, corrupt logs should not be a fatal
-    %% error.
-    NewStorage1 =
+reset_storage(Storage) ->
+    ets:delete_all_objects(Storage#storage.log_tab),
```

Comment:
Asking out of complete lack of knowledge on this:
1.  Why do we need to delete these now, but didn't need to do it before? 
2. What do the objects in those ets'es signify? 
3. We are deleting these ets entries for both the cases where the Sealer logs could be corrupt or not corrupt. Is that intended?

Reply:
1. Because these 2 ets tables are populated during parsing of the log files. You need to clean those tables if you want to start from the clean slate. We didn't need to erase these tables before, because we always started from newly created tables. 2. log_tab - log entries, log_info_tab - snapshot infos 3. It doesn't hurt to cleanup empty table.